contrib/kube-prometheus/README.md

```diff
@@ -374,7 +374,7 @@ If your kops cluster is using CoreDNS, there is an additional mixin to import.
 [embedmd]:# (examples/jsonnet-snippets/kops-coredns.jsonnet)
 ```jsonnet
 (import 'kube-prometheus/kube-prometheus.libsonnet') +
-(import 'kube-prometheus/kube-prometheus-kops.libsonnet') + 
+(import 'kube-prometheus/kube-prometheus-kops.libsonnet') +
```

Comment:
this is templated from `examples/jsonnet-snippets/kops-coredns.jsonnet`, so you need to remove the whitespace in there as well to make CI happy :slightly_smiling_face: